aip/aog/3001.md

```diff
@@ -7,9 +7,9 @@ aip:
   type: process
   scope: aog
 js:
-  - /assets/js/graphviz/viz.js
-  - /assets/js/graphviz/lite.render.js
-  - /assets/js/aip/aip-graphviz.js
+  - /static/js/graphviz/viz.js
+  - /static/js/graphviz/lite.render.js
+  - /static/js/aip/aip-graphviz.js
 permalink: /aog/3001
 redirect_from:
```

Comment:
Not having gotten all the way through the code yet: I'm hoping we are keeping the redirect ability.

Reply:
We are. It just no longer needs to be specified on an AIP by AIP basis.